src/consumer/index.js

```diff
@@ -172,12 +173,39 @@ module.exports = ({
     consumerGroup.seek({ topic, partition, offset })
   }
 
+  /**
+   * @returns Promise<GroupDescription>
```

Comment:
I like this a lot. We should do this for all places where we're currently just returning `Promise`.